libs/category/src/drivers/magento/transformers/category-page-config-transformer.service.ts

```diff
@@ -40,6 +40,19 @@ export class DaffMagentoCategoryPageConfigTransformerService {
     }
 	}
 
+	// There is no way to determine what type these filters are except through the
+	// Magento documentation.
+	private transformAggregateType(attribute_code: string): DaffCategoryFilterType {
```

Comment:
I think you should type `attribute_code` as `MagentoAggregation['attribute_code']`.

Reply:
done